modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/form.jsp

```diff
@@ -101,7 +101,7 @@ portletDisplay.setURLBack(backURL);
 			return fields.reduce((obj, field) => {
 				let value = field.value;
 				if (field.type === 'select' && !field.multiple) {
-					value = {key: field.value[0]};
+					value = {key: value.length === 0 ? '' : field.value[0]};
```

Comment:
You could only verify the `value.length` like this `key: value.length ? field.value[0] : ''` to make more simple, what do you think?

Reply:
If is really necessary to be `value.length === 0` could you explain me why? Thanks!

Reply:
Hi @rebsilva 

So, the bug occurs when value is empyt. So, when the user select _choose an option_ the value is "[]", it doesn't happen when the user select a field that exist key.  Makes sense to you? Thanks so much.

Reply:
Yeah that makes sense! The `key: value.length ? field.value[0] : ''` is actually verifying that: if the value is not empty, the key will be `field.value[0]`; otherwise it will be `''`. Its the same thing you did but without having to compare to zero you know? It also works if `value.length` is `undefined`.

Reply:
Perfect @rebsilva. Thanks